integration/benchmarks/benchmarks.yaml

```
@@ -16,39 +16,39 @@ tests:
     # trace only the client container
     gadgetParams: ["--containername=test-trace_dns-client"]
     server:
-      image: ghcr.io/mauriciovasquezbernal/bench
+      image: ghcr.io/inspektor-gadget/tools-bench:main
```

```suggestion
      image: ghcr.io/inspektor-gadget/benchmark:main
```

Same in other places...